src/main/java/com/zufar/icedlatte/product/entity/ProductInfo.java

```diff
@@ -1,11 +1,7 @@
 package com.zufar.icedlatte.product.entity;
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.Id;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Column;
-import jakarta.persistence.Table;
+import com.zufar.icedlatte.review.entity.ProductReview;
+import jakarta.persistence.*;
```

Comment:
nit: Evgeny suggested avoiding `*` imports, not sure why, though

Reply:
Fixed